moped-editor/src/views/projects/projectsListView/ProjectsListViewExportConf.js

```diff
@@ -67,6 +67,9 @@ export const PROJECT_LIST_VIEW_EXPORT_CONFIG = {
   project_status_update: {
     label: "Status Update",
   },
+  substantial_completion_date: {
+    label: "Substantial completion date",
+  },
   construction_start_date: {
     label: "Construction start",
     filter: formatTimeStampTZType,
```

Comment:
Last change - I swear! @tillyw could you update the config for this field to use this same filter so it shows as a date (not a timestamp) in the export file?

Reply:
yes, thanks for pointing that out!